docs/general/attribute-requirement-level.md

```diff
@@ -0,0 +1,78 @@
+# Attribute Requirement Levels
```

Comment:
We'll need to update all our internal links to the SPECIFICATION to this local document before you can delete the specification side, FYI.

That can be in this PR or a follow up.

Reply:
I've pushed a commit which updates all the references to the spec version of these documents I could find 👍